lib/coverband/collectors/delta.rb

```diff
@@ -12,7 +12,17 @@ def initialize(current_coverage)
 
       class RubyCoverage
         def self.results
-          ::Coverage.peek_result.dup
+          results = ::Coverage.peek_result.dup
+          Coverband.configuration.use_oneshot_lines_coverage ? transform_oneshot_lines_results(results) : results
+        end
+
+        def self.transform_oneshot_lines_results(results)
```

Comment:
I think want to keep our logic outside of this class. It's purpose is to allow for stubbing out ruby coverage for testing using dependency injection.

https://github.com/danmayer/coverband/blob/master/test/coverband/collectors/delta_test.rb

Reply:
yeah probably best to just put it in the primary Coverage collector class

Reply:
@danmayer Can you point me to the class you are referring to here as the "primary Coverage collector class".  Do you mean Coverband::Collectors::Delta?

Reply:
Think he means [coverage.rb](https://github.com/danmayer/coverband/blob/master/lib/coverband/collectors/coverage.rb)

Reply:
I moved the into the Delta class as that was my best guess at what you were suggesting

Reply:
@kbam, I didn't see your response until after I had pushed a commit.
I think the transform_oneshot_lines_results method needs to be used in the Delta class because it is comparing to the previous result which is also stored in the Delta class, so I left it there.

I can move it to Coverband::Collectors::Coverage if you that is the preference, but I think the usage would still stay in Delta.

Reply:
yeah I was thinking `collectors/coverage` let me give this PR another close look tomorrow. I should have a bit more time to dig in then